code/online_challenges/src/hackerrank/3D_aurface_area/3D_surface_area.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+
+int main()
+{
+    int h, w;
+    int a[110][110];
+    std::cin >> h >> w;
+    for (int i = 0; i < h; ++i) {
+        for (int j = 0; j < w; ++j)
+            std::cin >> a[i][j];
+    }
+    int sum = 0;
+    for (int i = 0; i < h; ++i) {
+        if (i == 0) {
+            for (int j = 0; j < w; ++j) {
+                int temp = 0;
+                if (j == 0) {
+                    temp = (a[i][j] - 1) * 4 + 6;
+                    sum = sum + temp;
```

Comment:
@aryabharat I mean for you to use compound assignment operators in all places in your code where possible (`+=`, `-=`, `*=`, etc.)

Reply:
ok, sir!